frontend/ui/uimanager.lua

```diff
@@ -876,8 +882,18 @@ function UIManager:sendEvent(event)
         end
     end
 
+    -- The top widget gets to be the first to get the event
     local top_widget = self._window_stack[#self._window_stack]
-    -- top level widget has first access to the event
+
+    -- A toast widget gets closed by any event, and
+    -- lets the event be handled by a lower widget
+    -- (Notification is our single widget with toast=true)
+    while top_widget.widget.toast do -- close them all
+        self:close(top_widget.widget)
```

Comment:
That *may* be problematic inside an actual `self._window_stack` walk, as it reorders the table.

UIManager:close() has to take some precautions because it *has* to do just that. Not *quite* sure how much we should care about it, though.

Reply:
Given that here, we don't really walk it: we always take `self._window_stack[#self._window_stack]`, so the one at the top of stack.
When we self:close() it, UIManger:close does walk from #self._window_stack to 1 - and should find it at top (as we found it at top), and remove it from top, and return.
Feels like no issue to me. To you?

Reply:
OK, didn't see your latest edit. So you agree with me :+1: